test/integration/forms/form.js

```diff
@@ -598,6 +598,12 @@ context('Patient Action Form', function() {
       .contains('Read Only')
       .should('be.disabled');
 
+    cy
```

Comment:
A simple test that verifies the form widgets container is shown and that it contains a few widgets.

Same test was also added for the patient form as well.